Upgrading to 2.0.1 breaks jenkins 2.2

I upgraded from 1.8.1 to 2.0.1 and it broke jenkins 2.2.

From the GUI

```
hudson.util.HudsonFailedToLoad: java.lang.AssertionError: class jenkins.plugins.slack.SlackNotifier$SlackJobProperty is missing its descriptor
    at hudson.WebAppMain$3.run(WebAppMain.java:240)
Caused by: java.lang.AssertionError: class jenkins.plugins.slack.SlackNotifier$SlackJobProperty is missing its descriptor
    at jenkins.model.Jenkins.getDescriptorOrDie(Jenkins.java:1301)
    at hudson.model.JobProperty.getDescriptor(JobProperty.java:105)
    at hudson.model.JobProperty.getDescriptor(JobProperty.java:79)
    at hudson.model.Descriptor.toMap(Descriptor.java:982)
    at hudson.model.Job.getProperties(Job.java:558)
    at org.jenkinsci.plugins.envinject.migration.EnvInjectMigrationListener.onLoaded(EnvInjectMigrationListener.java:45)
    at jenkins.model.Jenkins.<init>(Jenkins.java:917)
    at hudson.model.Hudson.<init>(Hudson.java:85)
    at hudson.model.Hudson.<init>(Hudson.java:81)
    at hudson.WebAppMain$3.run(WebAppMain.java:226)
```

/var/log/jenkins/jenkins.log

```
INFO: Jenkins stopped
Oct 13, 2016 11:58:54 AM hudson.init.impl.InstallUncaughtExceptionHandler$DefaultUncaughtExceptionHandler uncaughtException
SEVERE: A thread (Jenkins initialization thread/15) died unexpectedly due to an uncaught exception, this may leave your Jenkins in a bad way and is usually indicative of a bug in the code.
java.lang.AssertionError: class jenkins.plugins.slack.SlackNotifier$SlackJobProperty is missing its descriptor
        at jenkins.model.Jenkins.getDescriptorOrDie(Jenkins.java:1301)
        at hudson.model.JobProperty.getDescriptor(JobProperty.java:105)
        at hudson.model.JobProperty.getDescriptor(JobProperty.java:79)
        at hudson.model.Descriptor.toMap(Descriptor.java:982)
        at hudson.model.Job.getProperties(Job.java:558)
        at org.jenkinsci.plugins.envinject.migration.EnvInjectMigrationListener.onLoaded(EnvInjectMigrationListener.java:45)
        at jenkins.model.Jenkins.<init>(Jenkins.java:917)
        at hudson.model.Hudson.<init>(Hudson.java:85)
        at hudson.model.Hudson.<init>(Hudson.java:81)
        at hudson.WebAppMain$3.run(WebAppMain.java:226)

Oct 13, 2016 11:58:58 AM javax.jmdns.impl.DNSStatefulObject$DefaultImplementation waitForAnnounced
```

--- a/src/main/java/jenkins/plugins/slack/SlackNotifier.java
+++ b/src/main/java/jenkins/plugins/slack/SlackNotifier.java
@@ -479,7 +479,7 @@ public class SlackNotifier extends Notifier {
 
     }
 
-    @Extension public static final class Migrator extends ItemListener {
+    @Extension(ordinal = 100) public static final class Migrator extends ItemListener {
         @SuppressWarnings("deprecation")
         @Override
         public void onLoaded() {

